README.md

```diff
@@ -179,13 +179,15 @@ Unless otherwise noted these projects are open-source with permissive licensing:
 - [AI Sublime Text plugin](https://github.com/yaroslavyaroslav/OpenAI-sublime-text) (MIT)
 - [AIKit](https://github.com/sozercan/aikit) (MIT)
 - [LARS - The LLM & Advanced Referencing Solution](https://github.com/abgulati/LARS) (AGPL)
+- [GPUStack](https://github.com/gpustack/gpustack) (Apache-2.0)
```

Comment:
Should this be under the infrastructure category?

Reply:
thanks for pointing it out, updated.